apps/rush/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Change Log - @microsoft/rush
 
-This log was last generated on Fri, 21 Jul 2017 22:34:44 GMT and should not be manually modified.
+This log was last generated on Wed, 30 Aug 2017 18:46:19 GMT and should not be manually modified.
+
+## 3.0.15
+Wed, 30 Aug 2017 18:24:39 GMT
+
+### Patches
+
+- Add tar as a dependency of Rush which is required work for replacing the temp_modules/*/package.json files with TGZ files
```

Comment:
>eplacing the temp_modules/*/package.json files with TGZ files [](start = 62, length = 61)

This should be the feature announcement for end users.  (Who cares that we added "tar" as a dependency?)

Reply:
Changed the wording.

---
In reply to: [136167795](https://github.com/Microsoft/web-build-tools/pull/338#discussion_r136167795) [](ancestors = 136167795)